torchtune/modules/loss/cross_entropy_loss.py

```diff
@@ -79,19 +110,14 @@ def compute_cross_entropy(
 
     def forward(
         self,
-        weight: torch.Tensor,
         outputs: torch.Tensor,
```

Comment:
I guess outputs is a bit of a misnomer here? Do we have to keep it this way for BC?

Reply:
These are model outputs. I'm not sure what would be a better name. They used to be called logits, but they aren't logits in the linear loss case.